endless-sky/.devcontainer/docker-compose.yml

```diff
@@ -8,17 +8,18 @@ services:
     command: /bin/bash -c 'while sleep 1000;do true;done'
     #--device /dev/fuse --cap-add SYS_ADMIN --security-opt apparmor:unconfined
     devices:
-      - /dev/fuse
       - /dev/dri
+      - /dev/fuse
       - /dev/snd
     cap_add:
       - SYS_ADMIN
     security_opt:
       - 'apparmor:unconfined'
     volumes:
       - ../../../endless-sky:/workspace/endless-sky
-      - .:/workspace/endless-sky/.devcontainer
       - ../.vscode:/workspace/endless-sky/.vscode
+      - .:/workspace/endless-sky/.devcontainer
+      - ./clang-format:/workspace/endless-sky/.clang-format
```

Comment:
@quyykk this gets mounted inside of the developer container in the endless-sky source code root as `.clang-format`.